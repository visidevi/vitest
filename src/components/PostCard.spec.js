import { mount } from '@vue/test-utils'
import PostCard from './PostCard.vue'

describe('Post Card Component', () => {
  test('created posts renders correctly', () => {
    const title = 'First Post'
    const body = 'This is my first simple post'
    const wrapper = mount(PostCard, {
      props: { title, body },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
