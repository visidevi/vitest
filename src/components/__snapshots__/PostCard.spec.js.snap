// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Post Card Component > created posts renders correctly 1`] = `
"<div>
  <h1>First Post</h1>
  <p>This is my first simple post</p>
</div>"
`;
